package.json

```diff
@@ -435,6 +435,12 @@
 					"description": "Whether external libraries should be marked as debuggable.",
 					"scope": "resource"
 				},
+				"dart.evaluateGettersInDebugViews": {
+					"type": "boolean",
+					"default": true,
+					"description": "Whether to evaluate getters to display them in debug views (such as Variables, Watch and Hovers).",
```

Comment:
`Whether to evaluate getters in order to display them in debug views (such as the Variables, Watch and Hovers views).`?

Reply:
Done!

Reply:
+1 to shipping this by default with all flags turned on. We turned this on by default for dart4web users and never heard a single complaint about getters being evaluated.
We should consider adding a VM service protocol option to evaluate all getters to make this more efficient for cases where lots of objects are displayed.